docs/_documentation/using_comparison.md

```diff
@@ -12,50 +12,50 @@ In this section, we'll look at how Nakadi fits in with the stream broker/process
 
 - Pub/Sub uses a common envelope structure for producing and consuming messages, and does not define any higher level structures beyond that.
 
-### AWS Kinesis
+### Amazon Kinesis
 
 Like Nakadi and Pub/Sub, AWS Kinesis has a HTTP API to hide its details. Kinesis and Nakadi are more similar to each other than Pub/Sub, but there are some differences.
 
-- Kinesis expose shards (partitions) for a stream and supplies enough information to support per message checkpointing with semantics much like Kafka and Nakadi. Nakadi only supplies checkpointing information per batch of messages. Kinesis allows setting the partition hash key directly, whereas Nakadi computes the key based on the data. 
+- Kinesis exposes shards (partitions) for a stream and supplies enough information to support per-message checkpointing with semantics much like Kafka and Nakadi. Nakadi only supplies checkpointing information per batch of messages. Kinesis allows setting the partition hash key directly, whereas Nakadi computes the key based on the data.
 
-- Kinesis uses a polling model for consumers, whereas Nakadi maintains a streaming connection Kinesis consumers use a "shard iterator" to a grab pages of message, and then make a new HTTP request to grab another page. Kinesis limits the rate at which this can be done across all consumers (typically 5 transactions per second per open shard), which places an upper bound on consumer throughput. Kinesis has a broad range of choices for resuming from a position in the stream, Nakadi allows access only from the beginning and a named offset.
+- Kinesis uses a polling model for consumers, whereas Nakadi maintains a streaming connection. Kinesis consumers use a "shard iterator" to a grab pages of message, and then make a new HTTP request to grab another page. Kinesis limits the rate at which this can be done across all consumers (typically 5 transactions per second per open shard), which places an upper bound on consumer throughput. Kinesis has a broad range of choices for resuming from a position in the stream, Nakadi allows access only from the beginning and a named offset.
 
 - Kinesis uses a common envelope structure for producing and consuming messages, and does not define any higher level structures beyond that. Payload data is submitted as an opaque base64 blob.
 
-- AWS restrict the number of streams available to an account to quite a low starting number, and messages can be stored for a maximum of 7 days whereas Nakadi can support a large number of event types and the expiration for events is configurable.
+- Amazon restricts the number of streams available to an account to quite a low starting number, and messages can be stored for a maximum of 7 days, whereas Nakadi can support a large number of event types and the expiration for events is configurable.
```

Comment:
IMO ,`AWS` cannot be replaced by Amazon in this case. They are two different things

Reply:
Despite being a service offered in AWS, I think the product is indeed called [`Amazon Kinesis`](https://aws.amazon.com/kinesis/).